spec/takeaway_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  subject{Takeaway.new(sms_sender)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.